shared/AppInsightsCommon/src/applicationinsights-common.ts

```diff
@@ -58,7 +59,8 @@ export { IPropertiesPlugin } from "./Interfaces/IPropertiesPlugin";
 export { IUser, IUserContext } from "./Interfaces/Context/IUser";
 export { ITelemetryTrace, ITraceState } from "./Interfaces/Context/ITelemetryTrace";
 export { IRequestContext } from "./Interfaces/IRequestContext";
-export { eDistributedTracingModes, DistributedTracingModes } from "./Enums";
+export { IthrottleLocalStorageObj, IthrottleMgrConfig, IthrottleResult} from "./Interfaces/IThrottleMgr";
```

Comment:
Do we need to also export IThrottleLimit and IThrottleInterval?

We should if they will be referenced by using components (which I *think* they will)